test/g_searcher_web/controllers/search_result_controller_test.exs

```diff
@@ -0,0 +1,75 @@
+defmodule GSearcherWeb.SearchResultControllerTest do
+  use GSearcherWeb.ConnCase
+
+  describe "show/2" do
+    test "renders search result given user is logged in and belongs to user", %{conn: conn} do
```

Comment:
Maybe

```suggestion
    test "renders search result given search result belongs to user", %{conn: conn} do
```

and maybe for non-logged-in user in another test 💭

Reply:
I've changed the description, but for the test, I think testing non-logged in users on the `dashboard_controller` and plugs test might be enough? It feels duplicated for other controllers ©️